### ISSUE TITLE
Playlist issue

### ISSUE THREAD

**Issue by BigTruzThor** [vfsfitvnm/ViMusic#1853]
### Steps to reproduce the bug

1.go to any artist or user made playlist 
2.click on the playlist

### Expected behavior

It should show a normal working playlist 

### Actual behavior

It shows  an error

### Screenshots/Screen recordings

![Screenshot_20240806-173120](https://github.com/user-attachments/assets/c08ab4f7-3ec3-42e8-ab34-263cfcc5a152)

### ViMusic version

0.5.4

### Android version

Android 13

**Comment by mayankrockss** [comment]
Yes, was happening with previous  version of app as well. "Playlist you might like" playlist also yields same empty page.

**Comment by ItamiWasTaken** [comment]
Any issues you have with ViMusic will likely not be fixed, please refer to my post here: #1869
